### PR TITLE
Split input data on train and test parts

### PR DESCRIPTION
**Pull request by Bohdana-Zarembovska**
Split input data on train and test parts, renamed file

### CODE REVIEW

**Review by aASDa213ASD — APPROVED** [review]

File: kaggle_sets/data_preparation/train_test_split.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    x, y = dtnp(["date"], "mosquito_Indicator")
+"""
+
+
+class DatasetToNumpy:
+    def __init__(self, csv_file: str, csv_delimeter: str) -> None:
+        self.csv = Path(f"./datasets/{csv_file}.csv")
+        self.csv_delimeter = csv_delimeter
+
+    def __call__(self, drop_list: list, y_column: str, test_size: float = 0.2, random_seed: int = 42) -> tuple:
```
Comment (Zarathustra4) [inline kaggle_sets/data_preparation/train_test_split.py:18]: Will it work if we specify test_size = 0?

File: kaggle_sets/data_preparation/train_test_split.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+        drop_list.append(y_column)
+        x = dataframe.drop(drop_list, axis=1)
+        y = dataframe[y_column]
+        
+        np_x = x.to_numpy()
+        np_y = y.to_numpy()
+        np_y = np.expand_dims(y, axis=1)
+
+        np.random.seed(random_seed)
```
Comment (Zarathustra4) [inline kaggle_sets/data_preparation/train_test_split.py:32]: I suppose would be better to move your code into separate function and call it in ```_cast``` method

**Review by Zarathustra4 — CHANGES_REQUESTED** [review]

File: kaggle_sets/data_preparation/train_test_split.py (hunk 1 above)
Comment (Bohdana-Zarembovska) [reply kaggle_sets/data_preparation/train_test_split.py:18]: Yes

File: kaggle_sets/main.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-from data_preparation.numpy_cast import DatasetToNumpy
+from data_preparation.train_test_split import DatasetToNumpy
```
Comment (Zarathustra4) [inline kaggle_sets/main.py:1]: Rename module to "dataset_to_numpy"

Comment (Bohdana-Zarembovska) [reply kaggle_sets/main.py]: Done

File: kaggle_sets/data_preparation/train_test_split.py (hunk 2 above)
Comment (Bohdana-Zarembovska) [reply kaggle_sets/data_preparation/train_test_split.py]: Done

**Review by Zarathustra4 — APPROVED** [review]

**Pull request merged**